pkg/utils/utils.go

```diff
@@ -1126,22 +1128,37 @@ func IsValidUpgrade[T csmv1.CSMComponentType](ctx context.Context, oldVersion, n
 		return true, nil
 	}
 
-	// if not equal, it is an upgrade/downgrade
-	// get minimum required version for upgrade
-	minUpgradePath, err := getUpgradeInfo(ctx, operatorConfig, csmComponentType, newVersion)
+	var minUpgradePath string
+	var err error
+	var isUpgradeValid bool
+
+	log.Info("####oldVersion: ", oldVersion, " ###newVersion: ", newVersion)
+
+	isVersionCheck, _ := MinVersionCheck(oldVersion, newVersion)
```

Comment:
```suggestion
	isUpgrade, _ := MinVersionCheck(oldVersion, newVersion)
```
Apart from this, what do you think about renaming downgrade related variables e.g `isDowngradeValid`,  `maxDowngradePath`/`minDowngradePath`

Reply:
i guess isversioncheck should be good as its just veryfying if it should be upgrade or downgrade

Reply:
changed the variable names